### PR TITLE
remove CRVs experimental warning

### LINKED ISSUE #9780: Remove CRVs experimental label
We are close to meeting the requirements for . We should remove the experimental feature designation in the Rasa Open Source docs once they are met.

**Definition of done**
- [x] Remove experimental warning in docs
- [x] Remove the warning in the code
- [ ] Update the experimental feature table in Notion
- [ ] Let Dev Rel & Marketing know in release briefing

### PR DESCRIPTION
**Pull request by TyDunn**
**Proposed changes**:
- Closes #9780

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: docs/docs/responses.mdx [hunk 1]
```diff
@@ -140,22 +140,16 @@ in the shell and in interactive learning.
...
 response variation. A conditional response variation is defined in the domain or responses YAML files
 similarly to a standard response variation but with an additional `condition` key. This key specifies
 a list of slot `name` and `value` constraints.
 
 :::note
-Because the feature is currently experimental, it is recommended to set the `influence_conversation`
-slot setting to `False` for the slot mappings which are part of conditions.
+It is recommended to set the `influence_conversation` slot setting to `False` for the slot mappings
+which are part of conditions.
 :::
```
Comment (kedz) [inline docs/docs/responses.mdx:153]: Do we want to keep this recommendation? I have gotten several questions about it from CSEs and customers, and it seems to confuse people, and I just end up telling people that its just a recommendation and they can go ahead and try using featurized slots in CRV constraints. I haven't heard of anyone having a subsequent issue with that.

**Review by kedz — APPROVED** [review]
LGTM! I had one optional comment about removing a note from the docs but not critical to address here.

File: docs/docs/responses.mdx (hunk 1 above)
Comment (TyDunn) [reply docs/docs/responses.mdx:153]: @ancalita Wdyt?

Comment (ancalita) [reply docs/docs/responses.mdx:153]: I guess if no issues arose from using CRVs with featurized slots, it's fine to remove.

File: docs/docs/responses.mdx [hunk 2]
```diff
@@ -140,22 +140,16 @@ in the shell and in interactive learning.
...
 
 Specific response variations can also be selected based on one or more slot values using a conditional
 response variation. A conditional response variation is defined in the domain or responses YAML files
 similarly to a standard response variation but with an additional `condition` key. This key specifies
 a list of slot `name` and `value` constraints.
 
 :::note
-Because the feature is currently experimental, it is recommended to set the `influence_conversation`
-slot setting to `False` for the slot mappings which are part of conditions.
+It is recommended to set the `influence_conversation` slot setting to `False` for the slot mappings
```
Comment (TyDunn) [inline docs/docs/responses.mdx:151]:
```suggestion
```

File: docs/docs/responses.mdx [hunk 3]
```diff
@@ -140,22 +140,16 @@ in the shell and in interactive learning.
...
+which are part of conditions.
```
Comment (TyDunn) [inline docs/docs/responses.mdx:151]:
```suggestion
which are part of conditions.
```

File: docs/docs/responses.mdx [hunk 4]
```diff
@@ -140,22 +140,14 @@ in the shell and in interactive learning.
...
 
 Specific response variations can also be selected based on one or more slot values using a conditional
 response variation. A conditional response variation is defined in the domain or responses YAML files
 similarly to a standard response variation but with an additional `condition` key. This key specifies
 a list of slot `name` and `value` constraints.
 
 :::note
-Because the feature is currently experimental, it is recommended to set the `influence_conversation`
-slot setting to `False` for the slot mappings which are part of conditions.
 :::
```
Comment (TyDunn) [inline docs/docs/responses.mdx:151]:
```suggestion
```

File: docs/docs/responses.mdx [hunk 5]
```diff
@@ -140,22 +140,14 @@ in the shell and in interactive learning.
...
-us make this feature ready for production.
-:::
+### Conditional Response Variations
 
 Specific response variations can also be selected based on one or more slot values using a conditional
 response variation. A conditional response variation is defined in the domain or responses YAML files
 similarly to a standard response variation but with an additional `condition` key. This key specifies
 a list of slot `name` and `value` constraints.
 
 :::note
```
Comment (TyDunn) [inline docs/docs/responses.mdx:150]:
```suggestion
```

File: docs/docs/responses.mdx (hunk 4 above)
Comment (TyDunn) [inline docs/docs/responses.mdx:150]:
```suggestion
```

File: docs/docs/responses.mdx [hunk 6]
```diff
@@ -140,23 +140,13 @@ in the shell and in interactive learning.
...
 
 Specific response variations can also be selected based on one or more slot values using a conditional
 response variation. A conditional response variation is defined in the domain or responses YAML files
 similarly to a standard response variation but with an additional `condition` key. This key specifies
 a list of slot `name` and `value` constraints.
 
-:::note
-Because the feature is currently experimental, it is recommended to set the `influence_conversation`
-slot setting to `False` for the slot mappings which are part of conditions.
-:::
```
Comment (TyDunn) [inline docs/docs/responses.mdx:150]:
```suggestion
```

File: docs/docs/responses.mdx (hunk 1 above)
Comment (TyDunn) [reply docs/docs/responses.mdx]: Okay. I removed it

**Pull request merged**